src/main.rs

```diff
@@ -255,22 +255,24 @@ fn check_env_vars() {
     check_var::<String>("MEILISEARCH_ADDR");
     check_var::<String>("BIND_ADDR");
 
-    let storage_backend = &*dotenv::var("STORAGE_BACKEND")
-        .ok()
-        .unwrap_or("local".to_string());
+    check_var::<String>("STORAGE_BACKEND");
 
-    if storage_backend == "backblaze" {
+    let storage_backend = dotenv::var("STORAGE_BACKEND").ok();
+
+    if storage_backend == Some("backblaze".to_string()) {
```

Comment:
These should be `storage_backend.as_ref() == Some("backblaze")` to avoid the allocation;  Same for the other 2 comparisons